rllib/utils/tf_utils.py

```diff
@@ -570,7 +570,7 @@ def zero_logps_from_actions(actions: TensorStructType) -> TensorType:
 
 @DeveloperAPI
 def warn_if_infinite_kl_divergence(
-    policy: Type["TFPolicy"], mean_kl_loss: TensorType
+    policy: Type["TFPolicy"], mean_kl: TensorType
```

Comment:
curious! why renaming?

Reply:
the variable that gets passed to this function is called mean_kl_loss where it is used, however mean kl loss is computed like this:

```
action_kl = prev_action_dist.kl(curr_action_dist)
mean_kl_loss = reduce_mean_valid(action_kl)
```

so mean kl loss is just actually the mean kl. And this warning function is just checking to see that the mean kl is a finite number.

Reply:
it doesn't make sense to call it mean kl loss